client/src/IgocExplorer/IgocExplorer.js

```diff
@@ -90,72 +82,34 @@ const scheme = {
     ),
 };
 
-//This code is a little strange. This components exists as an intermediary between redux and react-router.
-//Trying to use a functional component here results in re-creating the redux store, connecter functions and Container component.
-//Instead, this component will own those long-term objects and keep the store updated with URL changes.
-class ExplorerContainer extends React.Component {
-  constructor(props) {
-    super();
-
-    const { grouping } = props;
-
-    const scheme_key = scheme.key;
-
-    const reducer = combineReducers({
-      root: root_reducer,
-      [scheme_key]: scheme.reducer,
-    });
-
-    const mapStateToProps = map_state_to_props_from_memoized_funcs(
-      get_memoized_funcs([scheme])
-    );
-
-    const mapDispatchToProps = (dispatch) => ({
-      ...map_dispatch_to_root_props(dispatch),
-      ...scheme.dispatch_to_props(dispatch),
-    });
-
-    const initialState = {
-      root: { ...initial_root_state, scheme_key },
-      [scheme_key]: {
-        grouping,
-        should_show_orgs_without_data: true,
-      },
-    };
+const IgocExplorer = ({ match }) => {
+  let grouping = _.get(match, "params.grouping");
```

Comment:
Should short circuit this, like `const grouping = _.get(match, "params.grouping") || "portfolio"`. Although now that I see it, this is copy and pasted from old code. Still should avoid it